src/components/StatsPanel/StatsPanel.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { useStatsPanel } from './StatsPanel.hook';
+import {
+  CardContainer,
+  StatItem,
+  StatIcon,
+  StatText,
+  StatValue,
+  IconWrapper,
+} from './StatsPanel.styles';
+
+import type { StatsPanelProps } from './StatsPanel.types';
+
+export const StatsPanel: React.FC<StatsPanelProps> = ({ totalTests, passed, failed, waiting }) => {
+  const stats = useStatsPanel({ totalTests, passed, failed, waiting });
+
+  return (
+    <CardContainer>
+      {stats.map((stat) => (
+        <StatItem key={stat.id}>
+          <IconWrapper>
+            <StatIcon color={stat.color}>{React.createElement(stat.icon)}</StatIcon>
+          </IconWrapper>
+          <div>
+            <StatValue>{stat.value}</StatValue>
```

Comment:
StatValue renders a raw number with no formatting or aria attributes. This is fine for simple cases but makes accessibility-based testing and localization harder.

**Sugerencia:**
```typescript
Consider adding an aria-label or data-testid to the value element and format numbers via Intl.NumberFormat if locale formatting is required. Example:

<StatValue aria-label={`stat-value-${stat.id}`}>{stat.value}</StatValue>
```